### ISSUE TITLE
Improve DgButton

### ISSUE THREAD

**Issue by jsmith** [dawg/dawg#170]
We need to improve the look/feel of our button component. Adam Wathan's "Refactoring UI" component gallery will serve as a good reference.

Maybe we could add back storybook for this? Maybe create a new repository? I hate the thought of doing this; however, these components would probably be fine living in a different repository.

**Comment by jsmith** [comment]
The files I'm referring to are the `Dg*.vue` files in `src/lib/essentials`. It's kinda a small component library in and of itself!

**Comment by heitorsilva** [comment]
@jsmith I think I should give a 2 cents here:

imho, having your UI components as a separate repo would only make sense if you plan to share UI elements among different projects, otherwise, it doesn't...

other use case would be providing the UI components as a library to the world, as a real other open-source project, that would have its own vision and goals.

think about it! ; )

**Comment by jsmith** [comment]
Yeah I'm thinking no to the separate repository thing but I still would like to improve this component and to maybe add storybook!
